app/routes.py

```diff
@@ -1,2 +1,205 @@
-from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from flask import request, Blueprint, make_response, jsonify
+from flask_sqlalchemy import SQLAlchemy
+import datetime
+from datetime import datetime, date, time, timezone
+from dotenv import load_dotenv
+import os
+import requests
+import json
 
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+
+@customers_bp.route("", methods=["GET"], strict_slashes=False)
+def customers_index():
+    
+    customers = Customer.query.all()
+    customers_response = []
+    
+    if customers is None:
+        return jsonify(customers_response), 200
+
+    else:
+        for customer in customers:
+            customers_response.append({
+                "id": customer.customer_id,
+                "name": customer.name,
+                "registered_at": customer.register_at,
+                "postal_code": customer.postal_code,
+                "phone": customer.phone,
+                "videos_checked_out_count": 0
+            })
+        return jsonify(customers_response), 200
```

Comment:
👍 this works perfectly! One thing to consider, though, is that the return statement is exactly the same, so let's save a line. Also, `query.all()` will always return a `[]` whether it is full of data or not, so we don't need to check if it's empty at first.  Either way, `customer_response` will be returned now:

```suggestion
    customers = Customer.query.all()
    customers_response = []
    
    for customer in customers:
        customers_response.append({
            "id": customer.customer_id,
            "name": customer.name,
            "registered_at": customer.register_at,
            "postal_code": customer.postal_code,
            "phone": customer.phone,
            "videos_checked_out_count": 0
        })
    return jsonify(customers_response), 200
```